lago/virt.py

```diff
@@ -964,6 +967,10 @@ def _create_live_snapshot(self, name):
                     )
                 )
 
+            self.wait_for_ssh()
+            self.guest_agent().start()
+            self.ssh('sync'.split(' '))
+
```

Comment:
It gives few more miliseconds for the logs to be written down, for the SSH to be ready, so 'sync' will do less work here and the SSH is going to wait a bit less. Probably benign.